File: TIC10002/src/main/java/uff/ic/lleme/tic10002/trabalhos/s20181/GabrielDuarte_CintiaCosta/Cliente.cpp
/*

	Gabriel Duarte
	Cintia Costa

*/

#ifndef CLIENTE
#define CLIENTE 1
#include <bits/stdc++.h>
using namespace std;

class Cliente{
private:
	int id, idade;
	string nome;
public:
	Cliente(){}
	Cliente(int _id, int _idade, string _nome){
		id = _id;
		idade = _idade;
		nome = _nome;
	}
	void setId(int _id){ id = _id; }
	void setIDade(int _idade){ idade = _idade; }
	void setNome(string _nome){ nome = _nome; }
	int getIdade(){ return idade; }
	string getNome(){	return nome;	}
	int getId(){ return id; }

	void imprimirCliente(){
		printf("Id: %d - Idade: %d - Nome: %s\n", id, idade, nome.c_str());
	}

};	
#endif

File: TIC10002/src/main/java/uff/ic/lleme/tic10002/trabalhos/s20181/Mateus_e_Raphael/src/trabalho_ed/Cliente.java
package trabalho_ed;

/**
 *
 * @author mateu
 */
public class Cliente {
  
    private int id = 0;
    private int idade = 0;
    private String CPF = null;
    private String nome = null;
    
    public Cliente(int id, int idade, String CPF, String nome){
        this.id = id;
        this.idade = idade;
        this.CPF = CPF;
        this.nome = nome;        
    }
    
    public int getIdade() {
        return idade;
    }
    
    public void setIdade(int idade) {
        this.idade = idade;
    }
    
    public int getId() {
        return id;
    }
    
    public void setId(int id) {
        this.id = id;
    }
    
    public String getCPF() {
        return CPF;
    }
    
    public void setCPF(String CPF) {
        this.CPF = CPF;
    }
    
    public String getNome() {
        return nome;
    }
    
    public void setNome(String nome) {
        this.nome = nome;
    }    
    
}
